static/js/analytics/support.js

```diff
@@ -0,0 +1,10 @@
+$(function () {
+    $("body").on("click", ".scrub-realm-button", function (e) {
+        e.preventDefault();
+        var string_id = $(this).data("string-id");
+        var message = 'Do you really want to scrub the realm "' + string_id + '"? This action is irreversible.';
```

Comment:
small thing, but I think the string_id part here isn't working (at least according to the screenshot).

Reply:
@rishig hmm. Can you elaborate? It seems to show the string_id ("realm05") correctly.

Reply:
ok, cool, I think I was looking at an old version.